app/mailers/twilio_sender.rb

```diff
@@ -2,23 +2,93 @@
 
 # TwilioSender: Methods to interact with Twilio REST API
 class TwilioSender
+  @twilio_error_codes = {
+    invalid_to_number: '21211',
+    blocked_number: '21610',
+    invalid_number: '21614',
+    unsupported_region: '21408',
+    unreachable_unavailable: '30003',
+    unavailable_ineligible: '30004',
+    non_existent_or_off: '30005',
+    sms_ineligible: '30006',
+    carrier_filter: '30007',
+    unknown_error: '30008'
+  }.freeze
+
   @client = Twilio::REST::Client.new(ENV['TWILLIO_API_ACCOUNT'], ENV['TWILLIO_API_KEY'])
-  def self.send_sms(patient, contents)
+  def self.handle_twilio_error_codes(patient, error_code)
```

Comment:
Why not combine these inline-defined error messages with the error code lookups from above, and define all of that in a YAML file like our other lookups.